src/styles/codeStyles/index.ios.js

```diff
@@ -1,15 +1,15 @@
 const codeWordWrapper = {
-    height: 20,
+    height: 22,
```

Comment:
Isn't 20 enough?

Reply:
In the case of 20, line-spacing is barely seen. Do you still want to set 20?

Reply:
Can you please attach two screenshots for 20 and 22

Reply:
**For 22**
![image](https://github.com/Expensify/App/assets/126839717/ac0bca6b-0632-4620-99ea-50fcb53dd5ad)

**For 20**
![image](https://github.com/Expensify/App/assets/126839717/b3eb022e-91bc-434f-9fad-91f0477c7038)

Reply:
@s77rt 
22 looks good IMO. What do you think?

Reply:
I think maybe 22 is slightly better, what do you think @s77rt ?

Reply:
@s-alves10 Can you please attach screenshots for Android?

Reply:
@s77rt 
Inline code blocks has a bug on Android devices as described [here](https://github.com/Expensify/App/issues/17441). This bug is out of the scope.

Reply:
Ah right.

Let's use the value that is closest to mWeb Safari. 22 seems too much difference. Should we keep it 20 or 21? (Just pick the one that match the mWeb Safari the most)

Reply:
No it's okay, not a blocker. We can go ahead with 22